praw/models/reddit/submission.py

```diff
@@ -492,7 +512,7 @@ def flair(self, text="", css_class=""):
         .. code:: python
 
            submission = reddit.submission(id='5or86n')
-           submission.mod.flair(text='PRAW', css_class='bot')
+           submission.mod.set_flair(text='PRAW', css_class='bot')
```

Comment:
Can we make `SubmissionsModerationFlair` callable so that we don't need to break this behavior?

Reply:
ahh, so the callable function runs the old one? got it.